example/otel-collector/otel-collector.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright The OpenTelemetry Authors
+# SPDX-License-Identifier: Apache-2.0
+
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:4317
```

Comment:
Do we need to specify this?

Reply:
https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/otlpexporter

Yes, as there is no default value for endpoint field.